packages/react-devtools-shared/src/__tests__/inspectedElement-test.js

```diff
@@ -2804,16 +2804,14 @@ describe('InspectedElement', () => {
       );
 
       // Suppress expected error and warning.
-      const originalError = console.error;
-      const originalWarn = console.warn;
-      console.error = () => {};
-      console.warn = () => {};
```

Comment:
These were fine because the test restored them. But if the test ever fails before it restores the methods, subsequent test will again not show console error|warn

Reply:
Yeah, good call. That restore should have been in an `afterEach` that checked test-scope level variables or something.